src/transformers/models/cohere2/modeling_cohere2.py

```diff
@@ -426,73 +430,6 @@ def _init_weights(self, module):
                 module.weight.data[module.padding_idx].zero_()
 
 
-COHERE2_INPUTS_DOCSTRING = r"""
```

Comment:
the diff in the docstring is not intentional.

I'm assuming that `modular` has seen logic changes since this file was last auto-generated (we skip models wo diff). If anyone knows how remove this from this PR's diff, lmk 🙏